web/Web/web/static/js/document-viewer.js

```diff
@@ -872,6 +872,11 @@ docView.prototype = {
         checkIfDocumentPreviouslyJudged(docid);
       }
 
+      let highlighted_keyword = Cookies.get('highlighted_keyword');
```

Comment:
Instead of having this be part of `document-viewer.js`, I think we should use `afterDocumentLoad ` instead.

It will be something like the code here. 

https://github.com/UWaterlooIR/gathera/blob/3f9e332bded6fc8fc4a6a1037894655795b2e485/web/Web/web/search/templates/search/search.html#L91-L104

Reply:
Have it be in a function in `highlight.js` and just call it in your `afterDocumentLoad` function.

Reply:
Resolved. Can you please check it again? Thank you!